test/functional/wallet_taproot.py

```diff
@@ -387,6 +390,14 @@ def do_test_psbt(self, comment, pattern, privmap, treefn, keys_pay, keys_change)
         psbt_online.unloadwallet()
         psbt_offline.unloadwallet()
 
+    def do_test_k_of_n(self, k, n):
+        self.do_test(
+            f"tr(XPUB,multi_a({k},XPRV_1,...,XPRV_N)",
+            f"tr($2/*,multi_a({k}" + (",$1/*" * n) + "))",
```

Comment:
Slightly unrelated: are there any tests that check multisig with different keys rather than repeating the same one, to make it more representative of a real-world scenario?
I'm just getting familiar with the codebase, I'm not sure whether it's important or not.

Reply:
I think it's very related and also my concern, is this test as good as one with K different keys. I'm hoping @Sjors can provide some insight here. My understanding of the other tests in this file is limited because they are not documented, but they seem to be doing something similar to this. On the other hand I could of course do what the previous test did, to use the "H_POINT" approach and just throw one XPRV in there somewhere. If that makes the test more real-world like or improves the test in some other way I'm all for it.

Reply:
Testing different keys makes sense, but let's keep this PR simple and leave it to a followup.